### DISCUSSION TITLE
mobile access to community server

### DISCUSSION THREAD

**Discussion by insomniux** [dani-garcia/vaultwarden#4284] [General]
Hi, excuse me if I've completely missed the concept, but I'm searching for a way to access my vault at the community server (normal browser access via public vaultwarden) using the vault/bitwarden app on my android phone. 
So far I do not have the plans (and possibility) to install my own server so hope this can be achieved using the community server.

**Comment by BlackDex** [comment]
This project does not have or maintain or is affiliated with any publicly (community) hosted Vaultwarden.
There will be no support given by this project for such publicly accessible servers.

If you want support on how to connect to a self-hosted instance see: https://bitwarden.com/help/change-client-environment/

**Reply by insomniux** [reply]
OK, thanks.
